### PR TITLE
add: landing page

### PR DESCRIPTION
**Pull request by Shivani-1524**
The Landing Page For the Ecommerce Site has been added.
Please give special attention to my gradient font code.
.new-font is temporary.

### CODE REVIEW

File: common.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+@import url("https://fonts.googleapis.com/css2?family=Audiowide&display=swap");
+
+.nav-logo img {
+  width: 35px;
```
Comment (sanjitsarkar) [inline common.css]: Use rem unit when something is greater than 10px as it makes it responsive.

File: index.css [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+}
+
+.new-font {
+  font-family: "Audiowide", cursive;
+}
+.content-container {
+  width: 100%;
+  height: 100%;
+}
+.gradient-font {
```
Comment (sanjitsarkar) [inline index.css:15]: Use variable for colors

**Review by sanjitsarkar — APPROVED** [review]

**Comment by eakansh125** [comment]
No issues found, just a suggestion:
image

When hovering on explore, a slight shift can be seen in the text. I tackled this issue by giving a transparent border/margin over the button, and during the hover, I gave it the same border/margin so the positions remained intact.

The page looks great, commendable job!

File: common.css (hunk 1 above)
Comment (Shivani-1524) [reply common.css]: I didnt want the image to scale but changed it anyway :)

File: index.css (hunk 2 above)
Comment (Shivani-1524) [reply index.css:15]: Do i have to use variables if im only going to use it at one place

**Comment by Shivani-1524** [comment]
> No issues found, just a suggestion: image
> 
> When hovering on explore, a slight shift can be seen in the text. I tackled this issue by giving a transparent border/margin over the button, and during the hover, I gave it the same border/margin so the positions remained intact.
> 
> The page looks great, commendable job!

Okay will change that! Thanks a lot 😄

**Pull request merged**